scripts/run_local.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+source venv/bin/activate
+export PYTHONPATH=$(pwd)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)